packages/cluster-ui/src/sortedtable/tableHead/tableHead.module.scss

```diff
@@ -1,31 +1,40 @@
 @import '../../core/index.module.scss';
 @import '../table.module.scss';
 
+@mixin table-header-text {
+  @include text--heading-6;
+  color: $colors--neutral-6;
+  font-weight: 600;
+  font-family: $font-family--semi-bold;
+  letter-spacing: 1.5px;
+}
+
 .head-wrapper {
   @include table-base;
 
-  &__cell--header {
+  .__cell--header {
```

Comment:
I know it's weird that we still have BEM in CSS modules, but as we are updating these styles can we lose the underscores? It's nitpicky and certainly not required for this change, but something to keep in mind as we refactor our styles.

Reply:
Agree, renamed to "cell-header" 👍🏼